scripts/assemble-weights.silva_132.sh

```diff
@@ -0,0 +1,107 @@
+# This workflow script contains all commands required to generate the
+#     taxonomic weights found in readytowear/data/silva/*/
+# Note that temporary files are removed at the end of this workflow.
+#     Remove the last line of this script to save temporary files.
+
+# move your current working directory to readytowear/ or alter this line
+pdir=.
+njobs=4
+
+cdir=$pdir/data/silva_132/
+mkdir $cdir
+
+tmp=$cdir/tmp/
+mkdir $tmp
+
+qiime rescript get-silva-data \
+    --p-version '132' \
+    --p-target 'SSURef_NR99' \
+    --p-include-species-labels \
+    --o-silva-sequences $tmp/silva-132-ssu-nr99-seqs.qza \
+    --o-silva-taxonomy $tmp/silva-132-ssu-nr99-tax.qza
+
+qiime rescript cull-seqs \
+    --i-sequences $tmp/silva-132-ssu-nr99-seqs.qza \
+    --o-clean-sequences $tmp/silva-132-ssu-nr99-seqs-cleaned.qza
+
+qiime rescript filter-seqs-length-by-taxon \
+    --i-sequences $tmp/silva-132-ssu-nr99-seqs-cleaned.qza \
+    --i-taxonomy $tmp/silva-132-ssu-nr99-tax.qza \
+    --p-labels Archaea Bacteria Eukaryota \
+    --p-min-lens 900 1200 1400 \
+    --o-filtered-seqs $tmp/silva-132-ssu-nr99-seqs-filt.qza \
+    --o-discarded-seqs $tmp/silva-132-ssu-nr99-seqs-discard.qza
+
+qiime rescript dereplicate \
+    --i-sequences $tmp/silva-132-ssu-nr99-seqs-filt.qza  \
+    --i-taxa $tmp/silva-132-ssu-nr99-tax.qza \
+    --p-rank-handles 'silva' \
+    --p-mode 'uniq' \
+    --o-dereplicated-sequences $cdir/full_length/ref-seqs.qza \
+    --o-dereplicated-taxa $cdir/full_length/ref-tax.qza
+
+
+# wget https://www.arb-silva.de/fileadmin/silva_databases/qiime/Silva_132_release.zip
+# unzip Silva_132_release.zip
+# qiime tools import --input-path $pdir/SILVA_132_QIIME_release/rep_set/rep_set_all/99/silva132_99.fna --type 'FeatureData[Sequence]' --output-path $cdir/full_length/ref-seqs.qza
+# qiime tools import --input-path $pdir/SILVA_132_QIIME_release/taxonomy/taxonomy_all/99/majority_taxonomy_7_levels.txt --output-path $cdir/full_length/ref-tax.qza --type 'FeatureData[Taxonomy]' --input-format HeaderlessTSVTaxonomyFormat
+# #cp $cdir/full_length/ref-tax.qza $cdir/515f-806r/ref-tax.qza
```

Comment:
We should reinstate or delete these lines. I recommend the former.

We can reinstate them because they work with the current version of QIIME 2. (They didn't used to work, see https://github.com/qiime2/q2-types/issues/165.)